### PR TITLE
Extract image features from SHSY5Y runs

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
In this PR, I extract image features from the CellProfiler SQLite output into its own parquet file. I also edit the documentation in the README and add this extraction into the "pipeline" of processing the CP features.

**Note:** I do not normalize or feature select any of this data.

### CODE REVIEW

File: Plate_2_data/4.processing_features/scripts/5.extract_image_features.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+# annotate df with platemap file to include all metadata
+annotated_image_features_df = annotate(
+    profiles=image_features_df,
+    platemap=platemap_df,
+    join_on=["Metadata_well_id", "Image_Metadata_Well"],
+    output_file="none",
+)
+
+# output df as parquet file
```
Comment (MikeLippincott) [inline Plate_2_data/4.processing_features/scripts/5.extract_image_features.py:97]: consider defining output path at beginning of file.

File: Plate_2_data/4.processing_features/scripts/5.extract_image_features.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+# In[4]:
+
+
+# extract image quality features from merged SHSY5Y runs image table df
+image_features_df = extract_utils.extract_image_features(
+    image_feature_categories=image_feature_categories,
+    image_df=SHSY5Y_run_df,
+    image_cols=image_cols,
+    strata=strata
```
Comment (MikeLippincott) [inline Plate_2_data/4.processing_features/scripts/5.extract_image_features.py:86]: could you document what strata is please?

File: Plate_2_data/4.processing_features/scripts/5.extract_image_features.py (hunk 1 above)
Comment (MikeLippincott) [reply Plate_2_data/4.processing_features/scripts/5.extract_image_features.py:97]: nevermind, it is dynamic

**Review by MikeLippincott — APPROVED** [review]
A couple of lines of documentation might need to be added but other than that, LGTM!

File: Plate_2_data/4.processing_features/scripts/5.extract_image_features.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+# Set file and directory constants
+cp_output_dir = pathlib.Path("../3.cellprofiler_analysis/analysis_output")
+features_output_dir = pathlib.Path("./data")
+platemap_df = pd.read_csv(pathlib.Path("../../metadata/Interstellar_plate2_platemap.csv"))
+
+# image categories/measurements to extract
+image_feature_categories = ["Image_Correlation", "Image_Granularity", "Image_Texture", "Image_Intensity"]
+image_cols="ImageNumber"
+strata=["Image_Metadata_Well", "Image_Metadata_Plate", "Image_Metadata_Site"]
```
[the remark this answers is not in the document]
Comment (jenna-tomkinson) [reply Plate_2_data/4.processing_features/scripts/5.extract_image_features.py:37]: No, in our case this is something that comes from Pycytominer and it is used for when you want to groupby/aggregate, but I use it to make sure all image identifiers are in the dataframe.

**Pull request merged**